sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -1376,7 +1371,7 @@ public void teardown() {
       // Set custom kafka serializers. We can not serialize user objects then pass the bytes to
       // producer. The key and value objects are used in kafka Partitioner interface.
       // This does not matter for default partitioner in Kafka as it uses just the serialized
-      // key bytes to pick a partition. But are making sure user's custom partitioner would work
+      // key bytes to pick a partition. But making sure user's custom partitioner would work
```

Comment:
I don't understand the last sentence, in the original nor rephrased version, can you rephrase it some more?